docs/current_docs/features/llm.mdx

```diff
@@ -1,39 +1,14 @@
 ---
-slug: /features/llm
+slug: /features/large-language-model
```

Comment:
Preference for the previous short URL :)

Reply:
That was a suggestion from @jasonmccallister for SEO